nixos/modules/tasks/filesystems/bcachefs.nix

```diff
@@ -239,6 +245,10 @@ in
         system.fsPackages = [ cfg.package ];
         services.udev.packages = [ cfg.package ];
 
+        boot.extraModulePackages = lib.optionals (!config.boot.kernelPackages.bcachefs.meta.broken) [
+          config.boot.kernelPackages.bcachefs
+        ];
```

Comment:
a small nit
```suggestion
        boot.extraModulePackages = lib.optional (
          !config.boot.kernelPackages.bcachefs.meta.broken
        ) config.boot.kernelPackages.bcachefs;
```

Reply:
There is no style guide on this, so there's no reason to nitpick it. IMO for a multiline expression, it's preferable for the lastmost subexpression to be the only multiline subexpression if possible. It simplifies how the expression is mentally processed IMO. Without any concrete reason why this is, I'm leaving it.

Reply:
@vladexa, @boomshroom, please **do not** make any style suggestions that are not either (a) in line with a documented policy (though feel free to work on introducing policies if you feel strongly about particular style decisions!) or (b) improve _egregiously_ unreadable code in really weird specific cases.

See also https://discourse.nixos.org/t/lib-optional-considered-harmful/7081

These reviews waste established contributors' time, discourage newbies by showing them that there are "rules" that they have to follow but can't just read up on, and worsen everyone's experience by drowning out valuable commentary.

Reply:
https://github.com/NixOS/nixpkgs/pull/445898 to codify this, because it's also silly to say this on a case-by-case basis.